docs/src/components/icons-list.astro

```diff
@@ -0,0 +1,71 @@
+---
+import { Icon } from '@astrojs/starlight/components';
+
+const icons = [
+  'up-caret',
+  'down-caret',
+  'right-caret',
+  'right-arrow',
+  'left-arrow',
+  'bars',
+  'translate',
+  'pencil',
+  'pen',
+  'document',
+  'add-document',
+  'setting',
+  'external',
+  'moon',
+  'sun',
+  'laptop',
+  'open-book',
+  'information',
+  'magnifier',
+  'forward-slash',
+  'close',
+  'error',
+  'warning',
+  'approve-check-circle',
+  'approve-check',
+  'rocket',
+  'star',
+  'puzzle',
+  'list-format',
+  'github',
+  'discord',
+  'twitter',
+  'mastodon',
+] as const;
+---
+
+<div class="icons-grid">
+  {
+    icons.map((icon) => {
+      return (
+        <div class="icon-preview">
+          <Icon name={icon} size="1.5rem" />
+          <span>{icon}</span>
+        </div>
+      );
+    })
+  }
+</div>
+
+<style>
+  .icons-grid {
+    display: grid;
+    gap: 1rem;
+    grid-template-columns: repeat(auto-fit, minmax(175px, 1fr));
```

Comment:
I think we can afford to have a narrower minimum width here (128px vs 175px). There’s one icon name that is too long and has to wrap, but I think that’s OK for having being able to show two columns on most mobile viewports.

```suggestion
    grid-template-columns: repeat(auto-fit, minmax(8rem, 1fr));
```